jina/peapods/zmq/__init__.py

```diff
@@ -385,29 +382,24 @@ def close(self, flush: bool = True, *args, **kwargs):
         :param kwargs: Extra key-value arguments
         """
         if not self.is_closed:
-            print(f'### is_closed was False {self.name}')
             self.is_closed = True
```

Comment:
If we remove the `force_close`, we need to remove this one as well.

Reply:
Why?